lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -228,14 +227,23 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
     $vars['show_persistent_signup'] = theme_get_setting('show_persistent_signup');
   }
 
+  // Get prepared Campaign data.
+  try {
+    $current_campaign = Campaign::get($vars['nid']);
```

Comment:
From what I've come to understand, if Drupal already loaded the node, it's cached and not reloaded so I figured it wouldn't do it harm. I was thinking about that too, but I think it's supposed to be ok.

Maybe "hydrating" is something to consider for a future refactor?